.env.example

```diff
@@ -52,3 +51,9 @@ MUGSHOT_BINARY_NPM=""
 MUGSHOT_NODE_MODULES_PATH=""
 MUGSHOT_BINARY_CHROME=""
 MUGSHOT_PUPPETEER_PROXY=""
+MUGSHOT_PUPPETEER_DISABLE_SANDBOX=false
+
+## Docker Settings
+
+FORWARD_APP_PORT=80
+FORWARD_MARIADB_PORT=3306
```

Comment:
Those variables need documentation. Just from that naming I cannot really tell what they are doing.

Reply:
now using the default values from sail with a hint